cypress/support/step_definitions/index.ts

```diff
@@ -213,6 +213,7 @@ Then(/^the "(.*)" element contains "(.*)"$/, (selector: string, value: string) =
 })
 
 Then('the page title contains {string}', function (title: string) {
+  cy.wait(1000)
```

Comment:
do we have like "waitUntil(title.contain(...))"?

Reply:
No, but we do have command timeouts. Unfortunately, in my experience, tests that are flaky related to this step definition don’t become less flaky with long command timeouts.

Reply:
I was more thinking about speeding this up not making it less flaky. Something like https://onsi.github.io/gomega/#eventually

Reply:
Cypress only works works under that semantic already. Each Cypress command has a built-in timeout (by default 4 seconds). So all our assertions _already_ "wait until assertion is true for 4  seconds".

Reply:
This will hopefully eventually be replaced by request counting i.e. after visiting a page don't proceed until all of our requests have responded i.e. when the client is idle. I 'think' this would remove the need for dropping waits everywhere.